core/arch/arch.c

```diff
@@ -3660,7 +3660,12 @@ dump_mcontext(priv_mcontext_t *context, file_t f, bool dump_xml)
     if (preserve_xmm_caller_saved()) {
```

Comment:
> No tests has been added as DynamoRIO itself does not clobber AVX-512 state. We explicitly currently enforce that DynamoRIO is compiled without AVX-512 for portability and unintended frequency scaling.

Wait: what about the client?  DR invokes client events at various points.

Reply:
See above.